docs/config/README.md

```diff
@@ -994,7 +994,7 @@ This is based on the `~/.config/gcloud/active_config` file and the `~/.config/gc
 | Option           | Default                                        | Description                                                     |
 | ---------------- | ---------------------------------------------- | --------------------------------------------------------------- |
 | `format`         | `'on [$symbol$account(\($region\))]($style) '` | The format for the module.                                      |
-| `symbol`         | `"☁️ "`                                        | The symbol used before displaying the current GCP profile.      |
+| `symbol`         | `"☁️ "`                                         | The symbol used before displaying the current GCP profile.      |
```

Comment:
Strangely, neither the old nor the new alignment seems to be correct here. Is this an artifact of how things render on web, or is the cloud emoji one of them weird double-width emojis that `wcswidth` miscalculates?

Reply:
That is just the way it is with emojis, they simply don't play well with monospaced fonts. Glitches like that are kinda expected.